### PR TITLE
DCOS_OSS-5562: added make_powershell function to 

### PR DESCRIPTION
**Pull request by sergiimatusEPAM**
## High-level description

What features does this change enable? What bugs does this change fix?
That one of steps to integrate the dcos_install.ps1 into dcos_generate_config.sh so that it will be deployed to Bootrsrtap machine
There is no packages changes introduced.

## Corresponding DC/OS tickets (required)

  - [DCOS_OSS-5562](https://jira.mesosphere.com/browse/DCOS_OSS-5562) Integrate Universal Installer to Winpanda setup process .

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain.
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]
  

@jongiddy , @DominikDary , please review/approve/merge. That's "easy" integration  we were talking on Architecture meeting, 10th of Oct 2019.

@mesosphere-mergebot , changelog-not-required Preparation to add Winpanda

### CODE REVIEW

**Comment by sergiimatusEPAM** [comment]
@mesosphere-mergebot request-review jongiddy

**Comment by sergiimatusEPAM** [comment]
@mesosphere-mergebot request-review dominikdary

File: gen/build_deploy/bash.py [hunk 1]
```diff
@@ -638,9 +638,23 @@ def validate_custom_check_bins_dir(custom_check_bins_dir):
...
     print("Generating Bash configuration files for DC/OS")
     make_bash(gen_out)
+    make_powershell(gen_out)
     util.do_bundle_onprem(gen_out, output_dir)
 
 
+def make_powershell(gen_out) -> None:
+    """Build powershell deployment script."""
+    # Populate the powershell script to Bootstrap for further deployment of Windows agent
+    powershell_script_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'powershell/dcos_install.ps1')
```
Comment (DominikDary) [inline gen/build_deploy/bash.py]: what about the `https://github.com/dcos/dcos/blob/master/gen/build_deploy/powershell/install_choco.ps1` file, isn't this necessary as well?

**Comment by sergiimatusEPAM** [comment]
> Is this PR supposed to include the `dcos_install.ps1` file? Otherwise the tests will not pass.

yes, @jongiddy, `powershell/dcos_install.ps1` has been included into the repository while yesterday's PR

**Comment by jongiddy** [comment]
@sergiimatusEPAM I think you need to add the PowerShell scripts to `setup.py`
https://github.com/dcos/dcos/blob/f4899e6d2dbf2a3a3a5d94c5da347796275852ef/setup.py#L123-L126

**Comment by sergiimatusEPAM** [comment]
> @sergiimatusEPAM I think you need to add the PowerShell scripts to `setup.py`
> https://github.com/dcos/dcos/blob/f4899e6d2dbf2a3a3a5d94c5da347796275852ef/setup.py#L123-L126

added the dcos_install.ps1 there. choco_install.ps1 is shipped along with pre-requisites zip

**Comment by jongiddy** [comment]
@mesosphere-mergebot changelog-not-required Introduction of preliminary Windows files

**Pull request merged**